Neos.Neos/Resources/Private/Templates/Module/Management/Workspaces/Show.html

```diff
@@ -115,9 +134,11 @@
 							var value = false;
 							if (jQuery(this).is(':checked')) {
 								value = true;
-								jQuery('.batch-action').removeClass('neos-hidden').removeClass('neos-disabled').removeAttr('disabled');
+								jQuery('.batch-action').removeClass('neos-hidden').removeClass('neos-discardAllChanges').removeClass('neos-disabled').removeAttr('disabled');
+								jQuery('.review-button-action').addClass('neos-hidden').addClass('neos-disabled').attr('disabled', 'disabled');
```

Comment:
Why do we call `addClass` and `removeClass` more than once here? Would be easier to read if we just write all classes in one addClass/removeClass call.
Also we should use `.prop()` for setting properties like `disabled`.

```js
jQuery('.batch-action').removeClass('neos-hidden neos-discardAllChanges neos-disabled').prop('disabled', false);
jQuery('.review-button-action').addClass('neos-hidden neos-disabled').prop('disabled', true);
```

Same for the other uses of jQuery below.

BTW: Do we still need jQuery here?

Reply:
> BTW: Do we still need jQuery here?

You're right, not really.

Ideally we should replace this with react or at least alpine js :D
But i think its more realistic to wait until someone decides to fully rewrite and reconceptualize the modules which use jQuery.

---------

JQuery stats of the Backend Modules:

litte bit: (easy to transform to vanilla)
Administration/Sites

solala: (still easy)
Management/History

50 lines spagetty: (intermediate)
Management/Workspaces